packages/expo-notifications/android/src/main/java/expo/modules/notifications/service/NotificationsService.kt

```diff
@@ -152,6 +163,46 @@ open class NotificationsService : FirebaseMessagingService() {
       })
     }
 
+    /**
+     * A helper function for dispatching a "get notification categories" command to the service.
+     *
+     * @param context Context where to start the service.
+     */
+    fun enqueueGetCategories(context: Context, receiver: ResultReceiver? = null) {
+      enqueueWork(context, Intent(NOTIFICATION_EVENT_ACTION, getUriBuilder().appendPath("categories").build()).also {
+        it.putExtra(EVENT_TYPE_KEY, GET_CATEGORIES_TYPE)
+        it.putExtra(RECEIVER_KEY, receiver)
+      })
```

Comment:
```suggestion
      enqueueWork(
        context,
        Intent(
          NOTIFICATION_EVENT_ACTION,
          getUriBuilder()
            .appendPath("categories")
            .build()
        ).also {
          it.putExtra(EVENT_TYPE_KEY, GET_CATEGORIES_TYPE)
          it.putExtra(RECEIVER_KEY, receiver)
        }
      )
```
It's only about formatting, but I believe splitting this method call into more lines might come in handy in terms of distinguishing separate params (especially this `Intent`-related things 😉 
And FWIW, I believe having a `builder` chained calls in separate lines is a must-have 😉 
Especially when there're more than one parameter to be set 😉 (see other comments)
Alternatively maybe we can lift this `Intent` creating up into local variable, but I like the inlined style more 😊